lending-pool/src/pool/bad_debt.rs

```diff
@@ -27,6 +26,9 @@ pub fn transfer_bad_debt_to_backstop(e: &Env, user: &Address) {
     let mut pool = Pool::load(e);
     let reserve_list = storage::get_res_list(e);
     let backstop_address = storage::get_backstop(e);
+    if user.clone() == backstop_address {
+        panic_with_error!(e, PoolError::BadRequest);
+    }
```

Comment:
can a test be added to exercise this?

Also, we should validate this before loading a bunch of data incurring a cost to the caller.